x-pack/test/functional/page_objects/index.ts

```diff
@@ -12,7 +12,7 @@ import { SecurityPageObject } from './security_page';
 import { MonitoringPageObject } from './monitoring_page';
 import { LogstashPageObject } from './logstash_page';
 import { GraphPageObject } from './graph_page';
-import { GrokDebuggerPageObject } from './grok_debugger_page';
+import { GrokDebuggerPageProvider } from './grok_debugger_page';
```

Comment:
Is `<appName>Provider` the correct naming pattern? I see a mix of that and `<appName>PageObject` in this file.

Reply:
I went ahead and changed this since I was already under the hood. There is a new convention for creating page objects that use a class rather than exporting a function. It's more for new tests being written going forward as there is no effort to convert the existing tests going on yet. Not sure if that's something we want to do for the rest of our tests but the GrokDebugger tests follow this convention. Great catch.